editor/project_manager.cpp

```diff
@@ -473,8 +473,20 @@ class ProjectDialog : public ConfirmationDialog {
 						cd->grab_focus();
 						return;
 					}
+					PackedStringArray project_features = ProjectSettings::get_required_features();
 					ProjectSettings::CustomMap initial_settings;
-					initial_settings["rendering/vulkan/rendering/back_end"] = rasterizer_button_group->get_pressed_button()->get_meta(SNAME("driver_name"));
+					// Be sure to change this code if/when renderers are changed.
+					int renderer_type = rasterizer_button_group->get_pressed_button()->get_meta(SNAME("driver_name"));
+					initial_settings["rendering/vulkan/rendering/back_end"] = renderer_type;
+					if (renderer_type == 0) {
+						project_features.push_back("Vulkan 1 Clustered");
+					} else if (renderer_type == 1) {
+						project_features.push_back("Vulkan 1 Mobile");
```

Comment:
I wouldn't put the "1" as it doesn't really matter (for now).

Reply:
I am somewhat concerned about what happens if we have a Vulkan 2.x renderer (whenever that comes into existence), but I guess then users will just know that no number means Vulkan 1.x.